<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainClass</class>
 <widget class="QMainWindow" name="AppMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>819</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AppMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="mainLayout"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupCamera">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Camera</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QComboBox" name="comboCamera">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>World</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Top</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Train</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupCurve">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Curve</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QComboBox" name="comboCurve">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cardinal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cubic</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupTrack">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Track</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QComboBox" name="comboTrack">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Track</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Road</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupTension">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="title">
         <string>Tension</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QSlider" name="sTension">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-50</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Play</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QPushButton" name="bPlay">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/AppMain/Resources/Icons/play.ico</normalon>
          </iconset>
         </property>
        </widget>
        <widget class="QSlider" name="sSpeed">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <zorder>bPlay</zorder>
        <zorder>sSpeed</zorder>
        <zorder>groupTension</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupCP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Control Point</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QPushButton" name="bAdd">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bDelete">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
        <zorder>bAdd</zorder>
        <zorder>bDelete</zorder>
        <zorder>groupPlay</zorder>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aLoadPath"/>
    <addaction name="aSavePath"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="aWorld"/>
    <addaction name="aTop"/>
    <addaction name="aTrain"/>
   </widget>
   <widget class="QMenu" name="menuCurve">
    <property name="title">
     <string>Curve</string>
    </property>
    <addaction name="aLinear"/>
    <addaction name="aCardinal"/>
    <addaction name="aCubic"/>
   </widget>
   <widget class="QMenu" name="menuTrack">
    <property name="title">
     <string>Track</string>
    </property>
    <addaction name="aLine"/>
    <addaction name="aTrack"/>
    <addaction name="aRoad"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuCurve"/>
   <addaction name="menuTrack"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aLoadPath"/>
   <addaction name="aSavePath"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="aLoadPath">
   <property name="icon">
    <iconset resource="AppMain.qrc">
     <normaloff>:/AppMain/Resources/Icons/openfile.ico</normaloff>:/AppMain/Resources/Icons/openfile.ico</iconset>
   </property>
   <property name="text">
    <string>Load Path</string>
   </property>
  </action>
  <action name="aSavePath">
   <property name="icon">
    <iconset resource="AppMain.qrc">
     <normaloff>:/AppMain/Resources/Icons/savefile.ico</normaloff>:/AppMain/Resources/Icons/savefile.ico</iconset>
   </property>
   <property name="text">
    <string>Save Path</string>
   </property>
  </action>
  <action name="aWorld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World</string>
   </property>
  </action>
  <action name="aTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="aTrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="aLinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="aCardinal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cardinal</string>
   </property>
  </action>
  <action name="aCubic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cubic</string>
   </property>
  </action>
  <action name="aLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="aTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </action>
  <action name="aRoad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Road</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppMain.qrc"/>
 </resources>
 <connections/>
</ui>
